src/modules/person/persistence/sponsor.entity.ts

```diff
@@ -0,0 +1,5 @@
+import { SchoolStructureNodeEntity } from './schoolStructureNode.entity.js';
+import { Entity } from '@mikro-orm/core';
+
+@Entity({ discriminatorColumn: 'nodeType', discriminatorValue: 'sponsor' })
+export class SponsorEntity extends SchoolStructureNodeEntity {}
```

Comment:
Not sure in which diagram this is still called sponsor, but the diagrams I can find it is called "school carrier".
You had other suggestions to name it, that I liked. If you can align on one with the PO/BA I'm happy to find a better term for it.